dev/dags/example_virtualenv.py

```diff
@@ -30,6 +30,9 @@
     profile_config=profile_config,
     execution_config=ExecutionConfig(
         execution_mode=ExecutionMode.VIRTUALENV,
+        # We can enable this flag if we want Airflow to create one virtualenv
+        # and reuse that within the whole DAG.
+        # virtualenv_dir=f"{get_airflow_home()}/persistent-venv",
```

Comment:
I think it is probably worth it for us to adapt this Example DAG doing the following:

1. Replace `DbtDag` by `DbtTaskGroup`
2. Have one `DbtTaskGroup` using `virtualenv_dir` to the python being used by the tests
3. Have one `DbtTaskGroup` with the current behavior

The advantage of changing the example DAG is that we would have an integration test validating this change and ensuring things keep working E2E for the two scenarios we care about.

An alternative to having a full `DbtTaskGroup` could be to call the Operator directly for the use-case when we'd reuse a previously existing virtualenv

Reply:
Good one, I've split them up :)